configure.ac

```diff
@@ -146,19 +146,20 @@ if test "$os_unix" = "yes"; then
 
 	AC_CHECK_FUNC(
 		[strerror_r],
-		[AC_RUN_IFELSE(
+		[AC_COMPILE_IFELSE(
 			[AC_LANG_SOURCE([[
```

Comment:
It would be slightly simpler to write this with [`AC_LANG_PROGRAM`](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.71/html_node/Generating-Sources.html#index-AC_005fLANG_005fPROGRAM).

Reply:
OK, I've rewritten it with `AC_LANG_PROGRAM`.